vscode-trace-extension/package.json

```diff
@@ -124,43 +124,56 @@
     },
     "views": {
       "trace-explorer": [
+        {
+          "type": "tree",
+          "name": "Welcome",
+          "id": "welcome",
+          "when": "trace-explorer.noExperiments || !traceViewer.serverOn"
+        },
         {
           "type": "webview",
           "id": "traceExplorer.openedTracesView",
-          "name": "Opened Traces"
+          "name": "Opened Traces",
+          "when": "!trace-explorer.noExperiments && traceViewer.serverOn"
         },
         {
           "type": "webview",
           "id": "traceExplorer.availableViews",
           "name": "Views",
-          "when": "!trace-explorer.noExperiments"
+          "when": "!trace-explorer.noExperiments && traceViewer.serverOn"
         },
         {
           "type": "webview",
           "id": "traceExplorer.timeRangeDataView",
           "name": "Time Range Data",
-          "when": "!trace-explorer.noExperiments"
+          "when": "!trace-explorer.noExperiments && traceViewer.serverOn"
         },
         {
           "type": "webview",
           "id": "traceExplorer.itemPropertiesView",
           "name": "Item Properties",
-          "when": "!trace-explorer.noExperiments"
+          "when": "!trace-explorer.noExperiments && traceViewer.serverOn"
         }
       ]
     },
     "viewsWelcome": [
       {
-        "view": "traces",
-        "contents": "No traces found [learn more](https://www.eclipse.org/tracecompass/).\n[Add traces](command:nodeDependencies.addEntry)"
+        "view": "welcome",
+        "name": "Start Server",
+        "contents": "The server is offline. \n[Resume Trace Extension](command:vscode-trace-server.start-if-stopped)",
```

Comment:
unfortunately, the `vscode-trace-extension` needs to be independent to the `vscode-trace-server` extension. There are users that don't use the `vscode-trace-server` extension and have their own way of starting the trace server. I suggest to look into the external API for starting a trace server. Right now a third-party extension (e.g. `vscode-trace-server`) can register their implementation to start the server. The current API requires a path to be passed. Maybe you can augment the API to add a default start api without passing the path. This should be optional.

Reply:
Looking at the `trace-server-manager` code I see a way to de-couple the starting of a trace server with opening a trace.  However, I will implement this into a separate PR.

For now, I'm simply going to remove the Start Server welcome view from this PR.  The code for this `viewsWelcome` contribution will be simple enough to re-introduce in the future.

Reply:
Thanks. Sounds good to split the PR.